src/main/java/bareum/Storage.java

```diff
@@ -0,0 +1,81 @@
+package bareum;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class Storage {
+    File storedTasks;
+
+    public Storage() {
+
+    }
+
+    public void loadSavedTaskList(TaskList taskList) {
+        File data = new File("./data");
+        if (!data.exists()) {
+            try {
+                data.mkdirs();
+            } catch (SecurityException e) {
+                System.out.println(e.getMessage());
+            }
+        }
+
+        File storedTasks = new File("./data/storedTasks.txt");
+        if (!storedTasks.exists()) {
+            try {
+                storedTasks.createNewFile();
+            } catch (IOException e) {
+                System.out.println(e.getMessage());
+            }
+        }
+        this.storedTasks = storedTasks;
+        StringBuilder savedTasks = new StringBuilder();
+        int currChar;
```

Comment:
Maybe the naming could be better? Since it's an integer

Reply:
Thanks for the feedback! I'll probably change it to something like `int currCharAsciiCode`